include/deal.II/matrix_free/operators.h

```diff
@@ -665,6 +665,25 @@ namespace MatrixFreeOperators
     apply(const VectorizedArrayType *in_array,
           VectorizedArrayType *      out_array) const;
 
+    /**
+     * This operation applies the inverse @ref GlossMassMatrix "mass matrix"
+     * operation on an input array with local tensor-valued coefficients.
+     * The second-rank tensor at each quadrature point defines a linear operator
+     * on a vector holding the dof components. It also uses the inverse of the
+     * JxW values provided by the `fe_eval` argument passed to the constructor
```

Comment:
Does the function use `JxW`? The comments seem to be very general and don't talk about the differences in comparison to the last function (scalar vs. tensorial coefficients).